entrypoint.sh

```diff
@@ -2,6 +2,7 @@
 
 set -euo pipefail
 
+./manage.py check --deploy
```

Comment:
My previous experience means I'm cautious about hardcoding this check into the *entrypoint*. If it fails, we then *cannot* start up the service. While that's technically valid and other apps do the same, practically it can be problematic. However, it's probably fine, given the low uptime situation we are in.

However, in case there's an easy way to do it, here's my reasoning:

There are are all kinds of reasons a why a check could give a false negative. There will be times when we want to just bring job-server anyway, even though we know a particular check has failed. Hardcoding it here makes that difficult, especially in a docker world.

Ideally, I'd prefer to run `manage.py check` at the same point that CHECKS is run, if dokku supports that.

Reply:
I definitely see your point here but didn't see a better place to put it 😬

CHECKS is effectively "curl $url, optionally check for $sting in response" unfortunately.

Reply:
Yeah. One of the thinks I've done before is to expose a url that 200s on successful check, and 500s on error, which could then be included in CHECKS

Reply:
I think I've seen that pattern talked about before too, with a `healthz` url.  Are you thinking we would effectively expose `python manage.py check --deploy` under this URL?